store/sqlite/sqlite.go

```diff
@@ -405,12 +405,24 @@ func (d db) DeleteGuestLink(id types.GuestLinkID) error {
 	return tx.Commit()
 }
 
-// Purge clears orphaned rows from the database.
+// Purge deletes expired entries and clears orphaned rows from the database.
 func (d db) Purge() error {
+	log.Printf("deleting expired entries from database")
+
+	_, err := d.ctx.Exec(`
+	DELETE FROM
+		entries
+	WHERE
+		entries.expiration_time IS NOT NULL AND entries.expiration_time < CURRENT_TIMESTAMP;`)
```

Comment:
Can we add a newline after the `AND` for consistency with our other SQL queries?